stock_reception_screen/models/stock_reception_screen.py

```diff
@@ -656,6 +656,53 @@ def button_save_step(self):
             return
         method = "process_{}".format(self.current_step)
         getattr(self, method)()
+        return True
+
+    def button_next_pack(self):
+        """Process the current package and prepare the screen to scan another
+        one for the same product/lot/packaging data.
+
+        When calling this button we are supposed to be in the `set_package` step,
+        and once triggered the button will complete the first steps with the
+        same data than the previous package, and will stop again on the
+        `set_package` step.
+        """
+        self.ensure_one()
+        if not self.current_move_id and not self.current_move_line_id:
+            return
+        assert self.current_step == "set_package", f"step = {self.current_step}"
```

Comment:
I didn't know that Python 3 f-string feature, nice.

Reply:
Available since Python 3.6 only!